test/verify/check-pages

```diff
@@ -312,11 +312,11 @@ OnCalendar=daily
         m.execute('echo scruffy:foobar | chpasswd')
 
         if "debian" in m.image:
-            m.execute('echo \'pt_BR.UTF-8 UTF-8\' >> /etc/locale.gen && locale-gen && update-locale')
+            m.execute('echo \'pt_BR.UTF-8 UTF-8\' >> /etc/locale.gen; locale-gen; update-locale')
```

Comment:
does `write()` have an append argument?  You split similar things like this out before, although I could understand if you don't want to do that here for reasons of symmetry.

Reply:
Indeed it does! I'll pick this up in the next round after we get that mkdir behaviour.

Reply:
Fixed in PR #17698